README.md

```diff
@@ -120,6 +120,8 @@ Python:
 * `-i, --ignore-args`: ignore args specified in the input dict (default = False)
 * `--offset [num]`: offset for arg file naming in chain jobs (default = 0)
 * `--maxJobs [num]`: max number of jobs to run (default = -1 -> not applied)
+* `-r, --resubmit`: resubmit the jobs based on where the input files are located (default = False)
```

Comment:
I would note this explicitly as a new mode of operation (à la getpy in https://github.com/cms-svj/SVJProduction)

Reply:
Done